Week-10/url-shortener/src/components/Header/style.css

```diff
@@ -0,0 +1,33 @@
+.headerContainer {
+  display: flex;
+  height: 80px;
+  width: 100%;
+  justify-content: space-between;
+  /* padg: 15px; */
+  align-items: center;
+  background-color: #232323;
+}
+
+.logo {
+  width: 90%;
+  font-size: 2rem;
+  font-weight: 700;
+  margin-left: 20px;
+}
+
+.links {
+  width: 10%;
+  margin-right: 30px;
+  display: flex;
+  justify-content: space-around;
```

Comment:
create a mixin at top level and reuse it across instead of writing same style in multiple css files